scripts/get-collectd-plugins-security-requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyYAML==5.1
```

Comment:
This file feels very hacky, can't we just upgrade the original plugin's requirements to match this?  You are forcing the plugins to run with these versions anyway so you might as well just change the plugin's requirements.txt to match and avoid this hack.

Reply:
You would also get the benefit of any testing in the plugin repos would be forced to use the newer versions to get deeper testing of them with these newer version.

Reply:
Yeah, wasn't sure how best to address this since we need to make sure that these versions are installed in case someone pins an earlier version in existing or new plugins.

Reply:
Ideally we just have any plugin that needs yaml has a requirement of `PyYAML >= 2.0` (or whatever the earliest version it works with is) and then when we you do the `pip install` it will just get the latest version.

A convenient way to see all the plugins' requirements.txt files is to just run a docker container in the layer right after the docker build has pulled the plugins from github.  Then you can see where the existing versions are coming from.

Also we probably don't want to pin to an exact version either, maybe let it use anything with the 5.x major release (for `PyYAML` and similar for others).